sr_multi_moveit/sr_multi_moveit_config/config/robot_configs/left_sh_ur10_biotac.yaml

```diff
@@ -15,3 +15,14 @@ robot:
           - up
       hand:
         name: shadowhand_left_motor_biotac.urdf.xacro
+
+ra_sr_ur_robot_hw:
+  # UR10 speed scale (0.0-1.0)
+  speed_scale: 0.5
+
+  # TODO: Hand E lite payload
+  payload_mass_kg: 4.55
+
+  # Hand H payload centre of inertia (estimated, need exact from Hardware)
```

Comment:
E

Reply:
Fixed in https://github.com/shadow-robot/sr_interface/pull/319/commits/7589dbe4a890140552b7a1012a72c3aecb595a1b.